pages/validators/hardware_requirements.md

```diff
@@ -2,10 +2,10 @@
 
 ## Minimum Specs
 
-The minimum recommended specs for running a node is the following:
+The minimum recommended specifications for running a node are the following:
 
 - 8-core, x86_64 architecture processor
-- 64 GiB RAM
-- 500 GiB of locally attached SSD storage
+- 64 GB RAM
+- 500 GB of locally attached SSD storage
```

Comment:
These are based on `GiB` not `GB`. We are referring to AWS specs https://aws.amazon.com/ec2/instance-types/